### DISCUSSION TITLE
1.3.2

### DISCUSSION THREAD

**Discussion by michael-r-elp** [EnderdracheLP/MultiplayerCore.Quest#15] [General]
## What's Changed
- Fixed entitlement checks #14 

**Full Changelog**: https://github.com/EnderdracheLP/MultiplayerCore.Quest/compare/1.3.1...1.3.2

We still like to ask any of you who try out the mod to please provide feedback.
That way we can know if the current release can become core again or should still be held back due to issues with some people crashing.

__*Feedback Form:*__ https://forms.gle/DwesiwAFUoAe9MDn6

<hr /><em>This discussion was created from the release <a href='https://github.com/EnderdracheLP/MultiplayerCore.Quest/releases/tag/1.3.2'>1.3.2</a>.</em>

**Comment by EduardoMaia** [comment]
When i fail, my screen becomes black, game freeze and i only can see/play again when the other player finish the song!
Fixed: Trick Saber mod is the problem. Removed and now all work perfect!
